app/models/manageiq/providers/google/network_manager/refresh_parser.rb

```diff
@@ -286,12 +281,17 @@ def get_load_balancer_pool_members(target_pools)
         @data[:load_balancer_pool_members] = []
 
         target_pools.each do |tp|
-          lb_pool_members = tp.instances.collect { |m| parse_load_balancer_pool_member(m) }
+          lb_pool_members = tp.instances.to_a.collect do |m|
+            @data_index.fetch_path(:load_balancer_pool_members, Digest::MD5.base64digest(m)) || parse_load_balancer_pool_member(m)
```

Comment:
Is it possible for both of these to be nil?  Should you `.compact` at the end of this?

Reply:
It shouldn't be nil, it can be an empty array though. I think the compact would be called when storing the inventory anyways.